packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ImageAndTextViewer/ImageAndTextViewerContainer.js

```diff
@@ -27,56 +32,43 @@ function ImageAndTextViewerContainer ({
   }
 
   const [mimeType] = Object.keys(subject.locations[frame])
-  const imageTypes = [
-    'image/png',
-    'image/jpeg',
-    'image/gif'
-  ]
 
   if (loadingState !== asyncStates.initialized) {
-    if (imageTypes.includes(mimeType)) {
-      return (
-        <Box
-          fill='horizontal'
-        >
-          <SingleImageViewer
-            enableInteractionLayer={false}
-            loadingState={loadingState}
-            onError={onError}
-            onReady={onReady}
-          />
-          <StepNavigation
-            onChange={handleFrameChange}
-            stepIndex={frame}
-            steps={[0, 1]}
-          />
-        </Box>
-      )
-    }
-    if (mimeType === 'text/plain') {
-      return (
-        <Box
-          fill='horizontal'
-          flex='grow'
-          justify='between'
-        >
-          <SingleTextViewer
-            onError={onError}
-            onReady={onReady}
-          />
-          <StepNavigation
-            onChange={handleFrameChange}
-            stepIndex={frame}
-            steps={[0, 1]}
-          />
-        </Box>
-      )
-    }
+    return (
+      <Box
+        fill='horizontal'
+      >
+        {(mimeType === 'text/plain')
+          ? (
+            <SingleTextViewer
+              height={`${dimensions[0]?.clientHeight}px`}
```

Comment:
This can be rendered as `height='undefinedpx'`, where `undefinedpx` isn't a valid CSS length.